### PR TITLE
Editorial pass

### PR DESCRIPTION
**Pull request by ShivanKaul**

### CODE REVIEW

File: draft-ietf-privacypass-protocol.md [hunk 1]
```diff
@@ -87,14 +84,14 @@ This information is covered in {{ARCHITECTURE}}.
 
 {::boilerplate bcp14}
 
-The following terms are used throughout this document.
+The following terms are used throughout this document.
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:87]:
```suggestion
The following terms are used throughout this document.
```

File: draft-ietf-privacypass-protocol.md [hunk 2]
```diff
@@ -87,14 +84,14 @@ This information is covered in {{ARCHITECTURE}}.
...
-The following terms are used throughout this document.
+The following terms are used throughout this document. 
 
-- Client: An entity that provides authorization tokens to services
-  across the Internet, in return for authorization.
-- Issuer: A service that produces Privacy Pass tokens to clients.
-- Private Key: The secret key used by the Issuer for issuing tokens.
-- Public Key: The public key used by the Issuer for issuing and verifying
-  tokens.
+- Client: Clients obtain Tokens from an Issuer and redeem them for authorization.
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:89]:
```suggestion
- Client: An entity that runs the Issuance protocol with an Issuer to produce Tokens that can be later used for redemption (see {{Section 2.2 of AUTHSCHEME}}).
```

File: draft-ietf-privacypass-protocol.md [hunk 3]
```diff
@@ -172,24 +169,26 @@ TokenChallenge ----> TokenRequest ------------->        |
...
-Issuers provide a Private and Public Key, denoted skI and pkI, respectively,
+Issuers provide a Private and Public Key, denoted `skI` and `pkI` respectively,
 used to produce tokens as input to the protocol. See {{issuer-configuration}}
-for how this key pair is generated.
+for how this key-pair is generated.
 
 Clients provide the following as input to the issuance protocol:
 
-- Issuer name, identifying the Issuer. This is typically a host name that
+- Issuer name: identifies the Issuer. This is typically a host name that
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:178]:
```suggestion
- Issuer name: An identifier for the Issuer. This is typically a host name that
```

File: draft-ietf-privacypass-protocol.md [hunk 4]
```diff
@@ -172,24 +169,26 @@ TokenChallenge ----> TokenRequest ------------->        |
...
-- Challenge value `challenge`, an opaque byte string. For example, this might
+- Challenge value: `challenge`, an opaque byte string. For example, this might
   be provided by the redemption protocol in {{AUTHSCHEME}}.
 
 Given this configuration and these inputs, the two messages exchanged in
 this protocol are described below. This section uses notation described in
 {{OPRF, Section 4}}, including SerializeElement and DeserializeElement,
 SerializeScalar and DeserializeScalar, and DeriveKeyPair.
 
+`Ne` and `Ns` are as defined in {{OPRF, Section 4}} for OPRF(P-384, SHA-384).
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:190]:
```suggestion
The constants `Ne` and `Ns` are as defined in {{OPRF, Section 4}} for OPRF(P-384, SHA-384).
```

File: draft-ietf-privacypass-protocol.md [hunk 5]
```diff
@@ -395,11 +393,11 @@ pair is generated.
 
 Clients provide the following as input to the issuance protocol:
 
-- Issuer name, identifying the Issuer. This is typically a host name that
+- Issuer name: identifies the Issuer. This is typically a host name that
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:396]:
```suggestion
- Issuer name: An identifier for the Issuer. This is typically a host name that
```

File: draft-ietf-privacypass-protocol.md [hunk 6]
```diff
@@ -524,7 +521,7 @@ If the rsabssa_finalize function fails, the Client aborts the protocol.
 
 ## Token Verification
 
-To verify a token, a verifier checks that Token.authenticator is a valid
+Tokens are verified by checking that Token.authenticator is a valid
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:524]:
```suggestion
Verifying a Token requires checking that Token.authenticator is a valid
```

File: draft-ietf-privacypass-protocol.md [hunk 7]
```diff
@@ -338,9 +336,9 @@ If the Finalize function fails, the Client aborts the protocol.
 
 ## Token Verification
 
-To verify a token, a verifier creates a VOPRF context using the Issuer Private Key,
-evaluates the token contents, and compares the result against the token authenticator
-value, as follows:
+Tokens are verified by creating a VOPRF context using the Issuer Private Key and Public Key,
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:339]:
```suggestion
Verifying a Token requires creating a VOPRF context using the Issuer Private Key and Public Key,
```

**Review by chris-wood — APPROVED** [review]

**Comment by chris-wood** [comment]
Thanks, @ShivanKaul! I applied some suggestions on top. I think this is an improvement, but will wait for some others to take a look, too.

File: draft-ietf-privacypass-protocol.md [hunk 8]
```diff
@@ -232,10 +231,10 @@ The structure fields are defined as follows:
 
 - "token_type" is a 2-octet integer, which matches the type in the challenge.
 
-- "truncated_token_key_id" is the least significant byte of the `token_key_id` in network byte order (in other words, the last 8 bits of `token_key_id`).
+- "truncated_token_key_id" is the least significant byte of the `token_key_id` ({{issuer-configuration}}) in network byte order (in other words, the last 8 bits of `token_key_id`).
 
 - "blinded_msg" is the Ne-octet blinded message defined above, computed as
-  `SerializeElement(blinded_element)`. Ne is as defined in {{OPRF, Section 4}}.
+  `SerializeElement(blinded_element)`.
```
Comment (chris-wood) [inline draft-ietf-privacypass-protocol.md:237]:
```suggestion
  `SerializeElement(blinded_element)`.
```

**Review by tfpauly — APPROVED** [review]

**Pull request merged**